pkg/dns-server/resolver/utils.go

```diff
@@ -0,0 +1,31 @@
+package resolver
```

Comment:
why `utils.go`? I'd put this next to the code where this is used. It's just a private function. Btw. it seems like it does not have to be bound to SimpleDNSResolver